networking/private-networking.html.md

```diff
@@ -10,115 +10,95 @@ redirect_from:
+Applications within the same organization are assigned special addresses (6PN addresses) tied to the organization. Those applications can talk to each other because of those 6PN addresses, but applications from other organizations can't; the Fly.io platform won't forward packets between different 6PN networks.
 
-This connectivity is always available to apps; you don't have to do anything special to get it.
+This connectivity is always available to apps by default; you don't have to do anything special to get it.
 
 You can connect apps running outside of Fly.io to your 6PN network using WireGuard; for that matter, you can connect your dev laptop to your 6PN network. To do that, you'll use flyctl, the Fly.io CLI, to generate a WireGuard configuration that has a 6PN address.
 
-## Discover apps through DNS on a Fly Machine
+## Fly.io `.internal` DNS
 
-An app's Fly Machines are configured with their DNS server pointing to `fdaa::3`. The DNS server on this address can resolve arbitrary DNS queries, so you can look up "google.com" with it. But it's also aware of 6PN addresses, and, when queried from a Machine, will let you look up the addresses of other apps in your organization. Those addresses live under the synthetic top-level domain `.internal`.
+A Fly Machine is configured to resolve domain names with a custom DNS server from the Fly Platform. This DNS server can resolve arbitrary DNS queries, so you can look up `google.com` with it. But it’s also aware of 6PN addresses, and will let you look up 6PN addresses for other apps in your organization. Those addresses live under the custom top-level domain `.internal`. You might want to use `.internal` domains to connect your app to databases, API servers, or other apps in your 6PN network.
 
-Since this is the default configuration we set up for Machines on Fly.io, you probably don't need to do anything special to make this work; if your app shares an organization with another app called `random-potato-45`, then you should be able to `ping6 random-potato-45.internal`.
+Underneath `.internal` there are second-level domains for every app in your Fly organization. For example, if your app is in an organization with another app called `random-potato-45`, then there will be a AAAA record at `random-potato-45.internal`. The AAAA record will contain *all* the 6PN addresses of the started Fly Machines that make up the `random-potato-45` Fly App. Note that different libraries and tools will use multi-address AAAA records differently; most will only use the first address that is returned, but others might round-robin between entries for every request -- if you'd like to know more, consult the documentation for the library or tool you are using for DNS lookup.
```

Comment:
Curious why we'd use `random-potato-45` instead of `my-app-name` (since we're tending to standardize this usage throughout the docs)?

Reply:
Ha, not my idea! It was already there as an example, so for consistency I extended its use throughout the page. But I didn't know there was an even higher standard for all the docs. I'll switch to `my-app-name`.